Source/Neptune.Web/Views/FieldVisit/FieldVisitSectionViewData.cs

```diff
@@ -0,0 +1,47 @@
+﻿using LtInfo.Common;
+using LtInfo.Common.Models;
+using Neptune.Web.Common;
+using Neptune.Web.Controllers;
+using Neptune.Web.Models;
+
+namespace Neptune.Web.Views.FieldVisit
+{
+    public class FieldVisitSectionViewData : NeptuneViewData
+    {
+        public Models.FieldVisit FieldVisit { get; }
+        public string FieldVisitInformationUrl { get; }
+        public string ScoreUrl { get; }
+        public string SectionName { get; }
+        public bool FieldVisitInformationComplete { get; }
+        public string SubsectionName { get; set; }
+        public string SectionHeader { get; set; }
+
+        public FieldVisitSectionViewData(Person currentPerson, Models.FieldVisit fieldVisit, Models.FieldVisitSection fieldVisitSection)
+            : base(currentPerson, StormwaterBreadCrumbEntity.FieldVisits)
+        {
+            FieldVisit = fieldVisit;
+
+            // todo
+            if (!ModelObjectHelpers.IsRealPrimaryKeyValue(fieldVisit.PrimaryKey))
```

Comment:
Thanks. I left this in case I needed to flesh it out when creating a new Field Visit but since that action is done on submission of a modal there doesn't need to be any conditional logic in the view data around it.